src/cart/swim/swim.c

```diff
@@ -1065,13 +1055,13 @@ swim_progress(struct swim_context *ctx, int64_t timeout)
 				D_GOTO(out, rc);
 			}
 			send_updates = false;
-		} else if (now + 100 < ctx->sc_next_event) {
+		} else if ((hlc + 100) < ctx->sc_next_event) {
```

Comment:
should this be  hlc + crt_msec2hlc(100) instead?

Reply:
Oh, yes. I miss this. :-(